### DISCUSSION TITLE
API key is invalid

### DISCUSSION THREAD

**Discussion by sonnylazuardi** [inlivedev/inlive-js-sdk#53] [Q&A]
anything need to do?

![image](https://user-images.githubusercontent.com/856609/217474175-06b0b0f0-71c4-4cfe-ae71-633501f48994.png)

**Comment by tyohan** [comment]
@sonnylazuardi could try to regenerate the API key and try it again? I'll try it again just in case some changes broke the things. Will back to you soon.

**Reply by tyohan** [reply]
We tried again our codepen example and it work as expected @sonnylazuardi. I suggest you to check the API key or regenerate it. Let us know if it still doesn't work for you.
